.github/workflows/build_and_test.yml

```diff
@@ -702,6 +702,8 @@ jobs:
       run: ./dev/mima
     - name: Scala linter
       run: ./dev/lint-scala
+    - name: Scala structured logging check
+      run: ./dev/structured-logging-style.py
```

Comment:
```
if [ -f ./dev/structured-logging-style.py ]; then
    ./dev/structured-logging-style.py
fi
```
`build_and_test.yml` will also be used for the daily test of branch-3.x, so we should make sure `./dev/structured-logging-style.py` exists before executing